cli/azd/CHANGELOG.md

```diff
@@ -1,14 +1,29 @@
 # Release History
 
-## 1.12.0-beta.1 (Unreleased)
+## 1.12.0 (2025-02-05)
 
 ### Features Added
```

Comment:
@vhvb1989 can you confirm if we should include https://github.com/Azure/azure-dev/pull/4750 or leave it out?

Reply:
Added it. Thank you